.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [14, 16, 18, 20]
+        node: [16, 18, 20, 22]
```

Comment:
We don't support Node 14, so removing. Also added Node 22 to CI since it's current and will enter active LTS soon.